jquery.nestable.js

```diff
@@ -243,31 +242,42 @@
                 .html(html);
         },
 
-        remove: function (itemId)
+        //use fade = 'fade' to fadeout item before removing.
+        remove: function (itemId, fade)
         {
-            var options = this.options;
-            var buttonsSelector = '[data-action="expand"], [data-action="collapse"]';
-
-            this._getItemById(itemId)
-                .remove();
-
-            // remove empty children lists
-            var emptyListsSelector = '.' + options.listClass
-                + ' .' + options.listClass + ':not(:has(*))';
-            $(this.el).find(emptyListsSelector).remove();
-
-            // remove buttons if parents do not have children
-            $(this.el).find(buttonsSelector).each(function() {
-                var siblings = $(this).siblings('.' + options.listClass);
-                if (siblings.length === 0) {
-                    $(this).remove();
-                }
-            });
-        },
+            var opts = this.options,
+                el   = this.el,
+                item = this._getItemById(itemId)
+
+            //removes item and additional elements from list
+            function removeItem(item){
+
+                // remove item
+                item = item || this;
+                item.remove();
+
+                // remove empty children lists
+                var emptyListsSelector = '.' + opts.listClass
+                    + ' .' + opts.listClass + ':not(:has(*))';
+                $(el).find(emptyListsSelector).remove();
+
+                // remove buttons if parents do not have children
+                var buttonsSelector = '[data-action="expand"], [data-action="collapse"]';
+                $(el).find(buttonsSelector).each(function() {
+                    var siblings = $(this).siblings('.' + opts.listClass);
+                    if (siblings.length === 0) {
+                        $(this).remove();
+                    }
+                });                
+            }
 
-        _getItemById: function(itemId) {
```

Comment:
@pjona, That's looks like my fault(I've several versions of nestable on my PC, where I changed location for this method, so here I've cut them and forgot to paste :(.... ). I will add another commit to this PR, today.